CMP1903_A1_2324/Testing.cs

```diff
@@ -3,18 +3,48 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Diagnostics;
 
 namespace CMP1903_A1_2324
 {
-    internal class Testing
+    public class Testing
     {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
-
-        //Method
+        public static void TestDieClass()
+        {
+            Die die = new Die();
+
+            // Test Roll method
+            //Debug.Assert(die.Roll() >= 1 && die.CurrentValue <= 6);
```

Comment:
Unsure if this is meant to still be here? If it is the comment would need to be removed and I would consider adding the second variable so it outputs some text describing what happens if the assert fails to meet the criteria.